src/lib/y2storage/dialogs/guided_setup/helpers/disk.rb

```diff
@@ -0,0 +1,94 @@
+# Copyright (c) [2019] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact Novell, Inc.
+#
+# To contact Novell about this file by physical or electronic mail, you may
+# find current contact information at www.novell.com.
+
+require "yast"
+require "y2storage"
+
+module Y2Storage
+  module Dialogs
+    class GuidedSetup
+      module Helpers
+        # Helper class to generate the label of a disk
+        class Disk
+          include Yast::I18n
+
+          # Constructor
+          #
+          # @param analyzer [Y2Storage::DiskAnalayzer]
+          def initialize(analyzer)
+            textdomain "storage"
+
+            @analyzer = analyzer
+          end
+
+          # Disk label used by dialogs
+          #
+          # The label has the form: "NAME, SIZE, [USB], INSTALLED_SYSTEMS".
+          #
+          # Examples:
+          #
+          #   "/dev/sda, 10.00 GiB, Windows, OpenSUSE"
+          #   "/dev/sdb, 8.00 GiB, USB"
+          #
+          # @return [String]
+          def label(disk)
+            data = [disk.name, disk.size.to_human_string]
+            data += type_labels(disk)
+            data += analyzer.installed_systems(disk)
+            data.join(", ")
+          end
+
+          private
```

Comment:
np: Are not we "out-denting" this `private` word?

Reply:
Yes, new version of rubocop prefers this way.

Reply:
well, in fact it is not that new version of rubocop prefers it, but new version has a bug that cause endless loop. For more details see comment in .rubocop.yml - https://github.com/yast/yast-storage-ng/blob/master/.rubocop.yml#L33

BTW no other module need it so far, as other modules does not have access modifier like 

```
  class A
  private
    ...
  end
```
which cause bug.

Reply:
Ah, ok. Thanks for the info. BTW, I personally prefer this version without out-denting.